pos_quick_logout/__manifest__.py

```diff
@@ -6,7 +6,7 @@
 
 {
     'name': 'Point of Sale - Quick Logout',
-    'version': '9.0.1.0.0',
+    'version': '1.0.0',
```

Comment:
please set '10.0.1.0.0'

Reply:
done in https://github.com/OCA/pos/pull/197/commits/262f4153487c2e792c8b07f066a2a2bcafe44d19